fulp_modules/main_features/bloodsuckers/code/bloodsucker/bloodsucker_coffin.dm

```diff
@@ -10,16 +10,16 @@
 	owner.teach_crafting_recipe(/datum/crafting_recipe/candelabrum)
 	owner.teach_crafting_recipe(/datum/crafting_recipe/meatcoffin)
 	// This is my Lair
-	coffin = claimed
-	lair = get_area(claimed)
-	to_chat(owner, "<span class='userdanger'>You have claimed the [claimed] as your place of immortal rest! Your lair is now [lair].</span>")
+	coffin = claiming
+	lair = get_area(claiming)
+	to_chat(owner, "<span class='userdanger'>You have claimed the [claiming] as your place of immortal rest! Your lair is now [lair].</span>")
 	to_chat(owner, "<span class='danger'>You have learned new construction recipes to improve your lair.</span>")
 	to_chat(owner, "<span class='announce'>Bloodsucker Tip: Find new lair recipes in the Misc tab of the <i>Crafting Menu</i>, including the <i>Persuasion Rack</i> for converting crew into Vassals.</span><br><br>")
 	return TRUE
 
 /// From crate.dm
 /obj/structure/closet/crate
-	var/mob/living/resident /// This lets bloodsuckers claim any "closet" as a Coffin.
+	var/mob/living/claimed_by /// This lets bloodsuckers claim any "closet" as a Coffin.
```

Comment:
Looking back at this, I think resident is better.

Reply:
I found it a bit confusing but I'll change it back, since "Resident" makes me think "Person currently inside the coffin"

Reply:
![image](https://user-images.githubusercontent.com/53777086/122685928-89369c80-d1dc-11eb-8602-46a8c0aa0c08.png)